session.go

```diff
@@ -398,5 +398,11 @@ func CreateSession(store kv.Storage) (Session, error) {
 
 	variable.BindSessionVars(s)
 	variable.GetSessionVars(s).SetStatusFlag(mysql.ServerStatusAutocommit, true)
+	b, ok := storeBootstrapped[store.UUID()]
```

Comment:
store.UUID may be not consistent, so if we restart tidb, we may re-run boostrap again.